www/js/controllers/favorite.js

```diff
@@ -0,0 +1,423 @@
+angular.module('app.controllers').controller('favorite', function ($scope, $timeout, socialActivity, homeCtrlParams,
+        profile, favorite, groups, flurry, $ionicScrollDelegate) {
+
+  flurry.log('news feed');
```

Comment:
sorry - that wasn't very clear (after rereading). The log says "news feed". should it say "favorites" ?